src/internal/utils.js

```diff
@@ -148,3 +148,24 @@ export function wrapSagaDispatch(dispatch) {
     return dispatch(wrappedAction)
   }
 }
+
+/**
```

Comment:
I don't really understand what you mean, should I remove the comment or format it differently ?